include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -165,11 +165,10 @@ NOTE(record_not_automatically_importable, none, "record '%0' is not "
 NOTE(projection_not_imported, none, "C++ method '%0' that returns unsafe "
                                     "projection of type '%1' not imported",
      (StringRef, StringRef))
-NOTE(dont_use_iterator_api, none, "C++ method '%0' that returns an unsafe "
-                                  "iterator not imported: use Swift Sequence "
-                                  "APIs instead",
+ERROR(dont_use_iterator_api, none, "'%0' is unavailable in Swift as it returns a C++ iterator. "
```

Comment:
Let us use a semicolon (;) instead of a fullstop to end this sentence. It seems to be the convention.